experiments/time_exploration_pandas.ipynb

```diff
@@ -0,0 +1,473 @@
+{
```

Comment:
This file can be added as a notebook in the examples folder that already exists. Please be sure to restart clear all